File: Loja de Games/Model/Jogo.php
<?php
class Jogo 
{
    public $id;
    public $nome;
    public $midia;
    public $genero;
    public $classificacao;
    public $estoque;
    public $imagem;

    public $con;

    public function __construct()
    {
        $conexao = new Conexao();
        $this->con = $conexao->retornarConexao();
    }

    public function Criar()
    {
        $sql = $this->con->prepare("INSERT INTO jogo (id, nome, midia, genero, classificacao, estoque, imagem) VALUES (null,?,?,?,?,?,?)");
        $sql->execute([$this->nome,$this->midia,$this->genero,$this->classificacao,$this->estoque, $this->imagem]);
    }

    public function Pesquisar($pesquisa)
    {
        if($pesquisa == "")
        {
            $sql = $this->con->prepare("SELECT * FROM jogo");
            $sql->execute();
            $row = $sql->fetchAll();
        }
        else
        {
            $sql = $this->con->prepare("SELECT * FROM jogo WHERE nome LIKE ? ");
            $sql->execute(['%'.$pesquisa.'%']);
            $row = $sql->fetchAll();
        }

        return $row;		
	}

	public function Listar()
    {
        $sql = $this->con->prepare("SELECT * FROM jogo");
        $sql->execute();
        $row = $sql->fetchAll();

        return $row;		
	}

    public function ListarUmJogo()
    {
        $sql = $this->con->prepare("SELECT * FROM jogo WHERE id = ?");
        $sql->execute([$this->id]);
        $row = $sql->fetchAll();

        if(sizeof($row) > 0)
        {
            return $row[0];
        }
        else
        {
            return $row;
        }		
	}

	public function Editar()
    {
        $sql = $this->con->prepare("UPDATE jogo SET nome = ?, midia = ?, genero = ?, classificacao = ?, estoque = ?  WHERE id=?");
		$sql->execute([$this->nome,$this->midia,$this->genero,$this->classificacao,$this->estoque,$this->id]);
	}

	public function Deletar()
    {
		$sql = $this->con->prepare("DELETE FROM jogo WHERE id=?");
		$sql->execute([$this->id]);
	}
}
?>

File: C#/loja_de_games/loja_de_games/Classes/Jogo.cs
using System.Data;

namespace loja_de_games.Classes
{
    public class Jogo
    {
        public int id { get; set; }
        public string nome { get; set; }
        public string midia { get; set; }
        public string genero { get; set; }
        public int classificacao { get; set; }
        public int estoque { get; set; }
        public string imagem { get; set; }
        public Conexao con { get; set; }

        public Jogo()
        {
            con = new Conexao();
        }

        public void Criar()
        {
            con.ExecutarComando($"INSERT INTO `jogo`(`id`, `nome`, `midia`, `genero`, `classificacao`, `estoque`, `imagem`) VALUES (null,{nome},{midia},{genero},{classificacao},{estoque},{imagem})");
        }

        public void Editar()
        {
            con.ExecutarComando($"UPDATE `jogo` SET `nome`= {nome},`midia`= {midia},`genero`= {genero},`classificacao`= {classificacao},`estoque`= {estoque},`imagem`= {imagem} WHERE id = {id}");
        }

        public void Deletar()
        {
            con.ExecutarComando($"DELETE FROM `jogo` WHERE id = {id}");
        }

        public DataTable ListarJogos()
        {
            return con.ExecutarConsulta($"SELECT * FROM `jogo`");
        }

        public DataTable Pesquisar(string pesquisa)
        {
            if (string.IsNullOrEmpty(pesquisa))
            {
                return ListarJogos();
            }
                    
            return con.ExecutarConsulta($"SELECT * FROM `jogo` WHERE `nome` = {pesquisa}");
        }
    }
}
